Assets/Editor/BuildWindow.cs

```diff
@@ -463,7 +469,21 @@ private class AndroidOperation {
       }
     }
 
-    private void OnBuildSettingsChanged() {
+    private void OnBuildSettingsChanged()
+    {
+      // Only set this if supporting SDK that needs Android (and is installed!).
+#if OCULUS_SUPPORTED
+#if UNITY_EDITOR_WIN
+      string adbExe = "adb.exe";
+#else
+      string adbExe = "adb";
+#endif
+      // If we're on Android cache the path to adb as it used during building. Need to do it pre-work so on main thread.
+      m_adbPath = BuildTiltBrush.GuiSelectedBuildTarget == BuildTarget.Android
+        ? Path.Combine(UnityEditor.Android.AndroidExternalToolsSettings.sdkRootPath, "platform-tools", adbExe)
```

Comment:
@billyquith , this line seems to break compilation for Rift, which is `OCULUS_SUPPORTED` but not Android. (at least in CI (Rift is not yet enabled, which is why it didn't fail here), where it doesn't appear to have the full Android tools installed).

```
Assets/Editor/BuildWindow.cs(483,24): error CS0234: The type or namespace name 'AndroidExternalToolsSettings' does not exist in the namespace 'UnityEditor.Android' (are you missing an assembly reference?)
```

We don't seem to have any unique defines for the platform, if I understand it correctly; that's all a runtime thing. We might need one, but if this is the only case, I'd hope we don't. I don't know enough C#, but is it possible to do something with, I dunno, reflection, to try loading this at run time rather than compile time?

cc @mikeskydev , assuming you're still working on a Rift build

Reply:
Actually, I think `#if UNITY_ANDROID` might be what we want. Testing now.

Reply:
yep, it builds now!